Library/Formula/vim.rb

```diff
@@ -74,11 +74,6 @@ def install
       opts << "--enable-luainterp"
     end
 
-    # XXX: Please do not submit a pull request that hardcodes the path
-    # to ruby: vim can be compiled against 1.8.x or 1.9.3-p385 and up.
-    # If you have problems with vim because of ruby, ensure a compatible
-    # version is first in your PATH when building vim.
```

Comment:
> inclusion of a depends_on :ruby should make it obvious enough hardcoding would be invalid, theoretically.

I confess it's not obvious to me, because when I see `depends_on` followed by a colon-prepended package name, I think, "not an ordinary ruby dep, but one that involves some conditional kung-fu depending on what my system is like, so maybe I can get HB to look at my special ruby location if I bug them".

In any case, the part of the section that says you can build from source against the first ruby in `$PATH` should be left in, because without that note, it really is totally unobvious.